ext/opentelemetry-ext-elasticsearch/src/opentelemetry/ext/elasticsearch/__init__.py

```diff
@@ -0,0 +1,155 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This library allows tracing HTTP elasticsearch made by the
+`elasticsearch <https://elasticsearch-py.readthedocs.io/en/master/>`_ library.
+
+Usage
+-----
+
+.. code-block:: python
+
+    import elasticsearch
+    import opentelemetry.ext.elasticsearch
+
+    opentelemetry.ext.elasticsearch.ElasticSearchInstrumentor().instrument()
+    es = elasticsearch.Elasticsearch()
+    es.index(index='my-index', doc_type='my-type', id=1, body={'my': 'data', 'timestamp': datetime.now()})
+    es.get(index='my-index', doc_type='my-type', id=1)
+
+API
+---
+
+# TODO: mention config params
+"""
+
+import functools
+import types
+from logging import getLogger
+from os import environ
+
+import elasticsearch
+import elasticsearch.exceptions
+from wrapt import ObjectProxy
+from wrapt import wrap_function_wrapper as _wrap
+
+from opentelemetry import context, propagators, trace
+from opentelemetry.auto_instrumentation.instrumentor import BaseInstrumentor
+from opentelemetry.ext.elasticsearch.version import __version__
+from opentelemetry.trace import SpanKind, get_tracer
+from opentelemetry.trace.status import Status, StatusCanonicalCode
+
+logger = getLogger(__name__)
+
+
+# Values to add as tags from the actual
+# payload returned by Elasticsearch, if any.
+_attributes_from_result = [
+    "found",
+    "timed_out",
+    "took",
+]
+
+_default_op_name = "request"
+
+
+class ElasticsearchInstrumentor(BaseInstrumentor):
+    """An instrumentor for requests
+    See `BaseInstrumentor`
+    """
+
+    def __init__(self, span_name_prefix=None):
+        if not span_name_prefix:
+            span_name_prefix = environ.get(
+                "OPENTELEMETRY_PYTHON_ELASTICSEARCH_NAME_PREFIX",
+                "Elasticsearch",
+            )
+        self.span_name_prefix = span_name_prefix.strip()
```

Comment:
`None` check?

Reply:
We set it to a default value on the above line when it is None